File: android/src/main/kotlin/com/cmc/c_shield_sdk/CShieldErrorCode.kt
package com.cmc.c_shield_sdk

internal object CShieldErrorCode {
    const val AIP_MISSING_HEADER    = "aip_missing_header"
    const val AIP_TIMESTAMP_EXPIRED = "aip_timestamp_expired"
    const val AIP_INVALID_SIGNATURE = "aip_invalid_signature"
    const val AIP_SIGNING_FAILED    = "aip_signing_failed"
    const val AIP_PROXY_CA          = "aip_proxy_ca"
    const val SSL_NOT_CONFIGURED    = "ssl_not_configured"
    const val SSL_PIN_MISMATCH      = "ssl_pin_mismatch"
    const val RASP_CHECKER_DISPOSED = "rasp_checker_disposed"
    const val NOT_INITIALIZED       = "not_initialized"
    const val INVALID_ARGUMENT      = "invalid_argument"
    const val NATIVE_ERROR          = "native_error"
}

File: ios/Classes/CShieldErrorCode.swift
import Foundation

internal enum CShieldErrorCode {
    static let aipMissingHeader    = "aip_missing_header"
    static let aipTimestampExpired = "aip_timestamp_expired"
    static let aipInvalidSignature = "aip_invalid_signature"
    static let aipSigningFailed    = "aip_signing_failed"
    static let aipProxyCA          = "aip_proxy_ca"
    static let sslNotConfigured    = "ssl_not_configured"
    static let sslPinMismatch      = "ssl_pin_mismatch"
    static let raspCheckerDisposed = "rasp_checker_disposed"
    static let notInitialized      = "not_initialized"
    static let invalidArgument     = "invalid_argument"
    static let nativeError         = "native_error"
}
